subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/buildcache/TaskOutputCachingNativePerformanceTest.groovy

```diff
@@ -17,11 +17,8 @@
 package org.gradle.performance.regression.buildcache
 
 import org.gradle.initialization.ParallelismBuildOptions
-import org.gradle.util.Requires
-import org.gradle.util.TestPrecondition
-import spock.lang.Unroll
 
-@Requires(TestPrecondition.LINUX)
+// TODO: Merge with TaskOutputCachingJavaPerformanceTest
```

Comment:
When shall we do this? Should there be an issue open for this?

Reply:
I created https://github.com/gradle/gradle/issues/14598.